source/getting-started/tutorials/sensors/index.md

```diff
@@ -0,0 +1,283 @@
+# Tutorial: Sensor Configuration, Feedback, & Profiling
+
+- **Goal:** Learn how to use the AMDC Timing Manager to get feedback from sensors
+- **Complexity:** 4 / 5
+- **Estimated Time:** TODO
+
+
+## Tutorial Requirements
+
+1. Working AMDC Hardware
+2. Completion of the [Hardware Commands](/getting-started/tutorials/hw-commands/index.md) tutorial
+3. Completion of the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial
+4. Completion of the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial
+
+This tutorial expands on the code created in the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial, and uses commands created in the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial. Both must be completed before this tutorial.
+
+## Introduction 
+
+In motor control applications, it is important to have consistent timing of both sensor data acquisition and the running of control tasks. In default configuration, the AMDC allows for some jitter between sensor measurement time and control loop timing. This tutorial shows how the AMDC can be configured to elimiate this jitter.
+
+This tutorial provides the source code for:
+* experimenting the effects of the timing manager using the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) app
```

Comment:
Does it make sense to have a bulleted list for a single item?

Suggested edit: "This tutorial provides code that allows users to experiment with the configuration of the AMDC Timing Manager. The effects of this experimentation will be observed through the lense of the previous [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial."